controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaAuthManager.java

```diff
@@ -127,6 +127,11 @@ public boolean authorize(String resource, Principal principal, String credential
         return handler.authorize(resource, principal).ordinal() >= level.ordinal();
     }
 
+    // For enabling unit testing
```

Comment:
Replace this comment with `@VisibleForTesting`

Reply:
Fixed.